src/HdbConfigurationManager.cpp

```diff
@@ -2338,8 +2337,9 @@ void HdbConfigurationManager::attribute_assign(const Tango::DevVarStringArray *a
 					(const char*)__func__, Tango::ERR);
 	}
 
-	bool removed = false;
-	for(archiver_map_t::iterator itmap = archiverMap.begin(); itmap != archiverMap.end(); itmap++)
+    bool removed = false;
```

Comment:
Bad identation

Reply:
Ah yes. It looks like the code is actually indented with tabs rather than spaces (my editor is set to insert spaces for tabs).

I will swap them for tabs, but perhaps at a ater date we can convert these to spaces so we do not hit these problems again.

Reply:
I have my editor set to indent with tabs .. :)

Reply:
Happy to stick with that here :) I suggested spaces as I have found the resulting layout both easier to maintain and work with on collaborative code/projects.

Reply:
Well, all main coding styles reccommend tabs for indentation... but you can always setup your editor to *show* them as two/four spaces then,

Reply:
In terms of coding styles I have seen both recommended in equal measure over the years. Its personal preference and standards you have been exposed to.  I have seen plenty of vim vs emacs style wars over it, so I tend to be happy to use what ever is required.

But just to illustrate why I suggested it. Look at libhdbpp-cassandra.h/cpp. Its now documented and I ran a formatter over it, the layout will only persist when using spaces. Tabs would move things about depending on the editor settings. Of course, it does not matter if, for example, streaming operators line up, but it does look nice...

Reply:
Code created by Pogo for C++ device servers uses tabs for indentation, so I would say there is no other choice in this case..